queries/man/injections.scm

```diff
@@ -0,0 +1,13 @@
+(section_heading
+  (section_title) @_title
+  (block) @injection.content
+  (#any-of? @_title "SYNOPSIS" "SYNTAX" "SINTASSI" "SKŁADNIA" "СИНТАКСИС" "書式")
```

Comment:
That's a bit of an arbitrary selection of languages. I'd prefer to stick to `en_US` only here.

Reply:
This will also include any "Feature Test Macro Requirements" lines.